frontend/src/lib/gql/gql-client.ts

```diff
@@ -61,6 +61,19 @@ function createGqlClient(_gqlEndpoint?: string): Client {
               if (args.input.orgId) {
                 cache.invalidate({__typename: 'OrgById', id: args.input.orgId}, 'projects');
               }
+              if (args.input.id) {
+                cache.invalidate({__typename: 'Project', id: args.input.id});
+                cache.invalidate({__typename: 'DraftProject', id: args.input.id});
+                // Note singular MyProject name for the myProjects query cache, ditto for draft
+                cache.invalidate({__typename: 'MyProject', id: args.input.id});
+                cache.invalidate({__typename: 'MyDraftProject', id: args.input.id});
```

Comment:
there's not actually a `MyProject` or `MyDraftProject` type  so those references are unneeded. Take a look at the schema.graphql file to see the types the exist. The `myProjects` query just returns a list of `Projects`

Reply:
And yet, when I comment out the `MyDraftProject` invalidation on line 69, draft projects don't show up in the "My Projects" immediately after clicking on the button. (They do show up on refresh). If I uncomment line 69, then the cache invalidation works as expected.

There's something going on with the typenames in the cache that isn't immediately obvious from looking at `schema.graphql`.